docs/getting-started/locally.md

```diff
@@ -0,0 +1,104 @@
+>  * Windows users should download the Windows Installer. Double-click and follow the prompts.
+
+## Installation
+
+Open a terminal (Applications > Utilities > Terminal on MacOS), and install nio using Python’s pip3 installer (verify the filepath and binary filename below):
+```
+pip3 install -U ~/Downloads/nio_lite-20180105-py3-none-any.whl
+```
+> MacOS Users: You may be prompted to install a dependency called "clang". This is a standard component that comes directly from Apple, so it's safe to click "OK" and re-run the command above.
+
+You can test that nio is correctly installed by running the following command:
+```
+which niod
+```
+> If you don’t see a path to niod, make sure Python’s binary directory is on your PATH.
+
+> Help setting your PATH: [Windows](https://msdn.microsoft.com/en-us/library/aa922003.aspx) / [MacOS](http://osxdaily.com/2014/08/14/add-new-path-to-path-command-line/).
+
+## Create a Project
+
+Projects are collections of services, blocks and configurations. Create a nio folder to keep things tidy:
+```
+mkdir -p nio/projects && cd nio/projects
+```
+Use the nio CLI (Command Line Interface) to create a new project named "my_project":
+```
+nio new my_project
+```
+> You'll be prompted for your Pubkeeper Hostname and Port. To locate these values:
+> * Open the **System Designer** in a browser: http://designer.n.io/
+> * Select your system in the left hand nav. (To create your first system, follow the instructions [here](https://docs.n.io/getting_started/in_the_cloud.html))
+> * Click the **Edit** button in the system toolbar to open its configuration.
+> * Enter the values for **hostname** and **token** when prompted.
+
+## Start your Project
+
+```
+cd my_project && niod
+```
+Log messages should display, similar to the following output. There should be no errors.
+
+```
+NIO [INFO] [main.WebServer] Server configured on 0.0.0.0:8181
+NIO [INFO] Component: ServiceManager status changed to: created
+NIO [INFO] Component: ServiceManager status changed from: created to: configuring
+NIO [INFO] Component: ServiceManager status changed from: configuring to: configured
+NIO [INFO] [main.WebServer] Starting server on 0.0.0.0:8181
+NIO [INFO] [main.WebServer] Server 0.0.0.0:8181 started on 0.0.0.0:8181
+NIO [INFO] Component: ServiceManager status changed from: configured to: starting
+NIO [INFO] Component: ServiceManager status changed from: starting to: started
+```
+
+If you see those logs, the nio Platform is up and running. Congratulations!
+
+> By default, new projects use port 8181. If port 8181 isn’t available, you'll see an error. Change the NIOPORT value in your project's nio.env file to fix this error.
+
+> **MacOS Users**: Python 3.6.4+ requires that you install and trust a set of Root Certificates for Python's SSL package. That file is located at /Applications/Python 3.6/Install Certificates.command. Just double-click that file to complete the process. Check /Applications/Python 3.6/ReadMe.rtf for more details.
```

Comment:
As we've seen, it's not always Python 3.6.4+. I think it more has to do with non-native Python installs. We should clarify and also include an example of the errors that are seen when you need to install the root certs

Reply:
changed the wording